apps/examples/board_specific_demo/ameba_wifi_csi_demo/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_WIFICSI
+	bool "WIFI CSI example"
+	default n
+	depends on CONFIG_SE_AMEBA

```

Comment:
Is this related with SE? Could you let me know the reason?

Reply:
Hi Mr Chang,
This example is not related to SE, sorry it was a typo, fixed it. Thanks